<ui version="4.0" >
 <class>SensorLoggerSettingsWidget</class>
 <widget class="QWidget" name="SensorLoggerSettingsWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>249</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="titleFrame" >
     <property name="title" >
      <string>Title</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <item>
       <widget class="KLineEdit" name="m_title" />
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorFrame" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="title" >
      <string>Colors</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>11</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="textLabel1" >
          <property name="text" >
           <string>Text color:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel2" >
          <property name="text" >
           <string>Background color:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="textLabel3" >
          <property name="text" >
           <string>Alarm color:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="wordWrap" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <item>
         <widget class="KColorButton" name="m_foregroundColor" >
          <property name="text" >
           <string/>
          </property>
          <property name="color" >
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="m_backgroundColor" >
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KColorButton" name="m_alarmColor" >
          <property name="text" >
           <string/>
          </property>
          <property name="color" >
           <color>
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer" >
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
